.circleci/config.yml

```diff
@@ -38,9 +38,35 @@ defaults: &defaults
 
 version: 2
 jobs:
+    configure_env_vars:
+        <<: *defaults
+
+        docker:
+            - image: quay.io/pantheon-public/build-tools-ci:visual-regression-vars
```

Comment:
This needs to be `quay.io/pantheon-public/build-tools-ci:4.x` after https://github.com/pantheon-systems/docker-build-tools-ci/pull/19 is merged/deployed